include/rdma/providers/fi_peer.h

```diff
@@ -169,6 +169,7 @@ struct fi_peer_rx_entry {
 	uint64_t tag;
 	uint64_t cq_data;
 	uint64_t flags;
+	uint64_t ignore;
```

Comment:
Separate change. Also not sure why this is necessary? Only the owner should need the ignore bits for matching. The peer_rx_entry is only for fields that need to be shared between owner and peer.

Reply:
moved to owner structure